pymc3/step_methods/hmc/quadpotential.py

```diff
@@ -342,6 +360,8 @@ def __init__(
 
     def add_sample(self, x, weight):
         x = np.asarray(x)
+        if weight != 1:
+            raise ValueError("weight is unused and broken")
```

Comment:
```suggestion
            raise ValueError("Setting weight != 1 is not supported.")
```

Or maybe we should just remove it all-together.

Reply:
done